locale/he/locale.cfg

```diff
@@ -24,14 +24,25 @@ sensor.player_locations.err_no_name=.םיפתתשמ הבורמ קחשמ לרטנ
 sensor.player_locations.surface_fragment=__1__ לע
 sensor.player_locations.offline_fragment=]קתונמ[
 sensor.player_locations.settings.title=ןקחש ימוקימ תורדגה
-sensor.player_locations.settings.show_player_index=ןקחש רפסמ הארה
-sensor.player_locations.settings.show_position=ןויצ תודוקנ הארה
-sensor.player_locations.settings.show_surface=וילע ןקחשהש םוקמה תא הארה
-sensor.player_locations.settings.show_direction=ןקחשל ןוויכ הארה
+sensor.player_locations.settings.show_player_index=ןקחש רפסמ
+sensor.player_locations.settings.show_position=ןויצ תודוקנ
+sensor.player_locations.settings.show_surface=וילע ןקחשהש םוקמה תא
+sensor.player_locations.settings.show_direction=ןקחשל ןוויכ
+sensor.player_locations.settings.show_offline=םיקתונמ םינקחש
 
 sensor.pollution_around_player.name=הבורקה הביבסב םוהיז תמר
 sensor.pollution_around_player.format=PU. __1__ :םוהיז תמר
 
+sensor.kill_count.name=(הצובקל) תוגירה
+sensor.kill_count.format=Kills: __1__, __2__, __3__.
+sensor.kill_count.biter_fragment_single=םינכשנ 1
+sensor.kill_count.biter_fragment_multiple=םינכשנ __1__
```

Comment:
Looks like the placeholder `__1__` got broken here and on subsequent lines, as half of it seems to show up at the end of the line instead of next to the digit 1. Can you please confirm that you've tested this, and it works?